src/main/java/me/eccentric_nz/TARDIS/placeholders/TARDISPlaceholderExpansion.java

```diff
@@ -118,12 +119,10 @@ public String onPlaceholderRequest(Player player, String identifier) {
                         default:
                             // user
                             if (split.length > 2) {
-                                OfflinePlayer offlinePlayer = plugin.getServer().getOfflinePlayer(split[2]);
-                                if (offlinePlayer != null) {
-                                    rsti = new ResultSetTardisID(plugin);
-                                    if (rsti.fromUUID(offlinePlayer.getUniqueId().toString()) && rsti.getTardis_id() == rsv.getTardis_id()) {
-                                        result = "true";
-                                    }
+                                OfflinePlayer offlinePlayer = plugin.getServer().getOfflinePlayer(UUID.fromString(split[2]));
```

Comment:
Why do we need to get an offline player at all if we're just using it to convert the player UUID back to a string again? Surely `split[2]` is equal to `offlinePlayer.getUniqueId().toString()` in this case...

Reply:
In this specific placeholder, I believe is `%tardis_in_user_<username>%`, split[2] would be the other player in this case, not the offline player in question. We're comparing OfflinePlayer in question to see if they're in said user's TARDIS.